components/dimmed/Dimmed.tsx

```diff
@@ -0,0 +1,11 @@
+import './Dimmed.css';
+
+interface Props {
+  onClick?: (e: React.MouseEvent<HTMLElement>) => void;
+}
```

Comment:
```suggestion
import type { ComponentProps } from 'react';

interface Props extends ComponentProps<'div'> {}
}
```

이렇게 하면 div가 가진 모든 props를 활용할 수 있습니다.